dopedao/dope-monorepo
# Unbundle loot without minting a hustler

From here https://dope-wars.notion.site/f6e68234cb3145f3a02ae8d215e9f63f?v=762e20115798400891f1ee68a0229636&p=abbc935203b349e8bd6e1a5fac0dea2f

Duplicated the "Initiate" and "Mint Success" hustler pages and customized them for the context of unbundling loot instead.

We should consider refactoring the code to reuse the components if we want the UI's of these to be so similar.

I don't have any code currently checking if the loot is already unbundled. Someone can hit the loot unbundle url directly and attempt to do so but I think it fails safely.

Manual testing:
- Approving paper works
- Unbundling sends call through metamask fine but I don't know how to verify
- Unbundling and sending to another address appears to work  but I don't know how to verify
- After unbundle goes to gangsta party page for loot and can click to go back to loot page
